sources/bloodhound/src/output.rs

```diff
@@ -56,3 +78,156 @@ impl ReportWriter for JsonReportWriter {
         writeln!(output, "{json}")
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::results::{CheckStatus, CheckerMetadata, CheckerResult, Mode, ReportMetadata};
+    use std::io::Cursor;
+
+    #[test]
+    fn test_text_output_override_pass_fail_statuses() {
+        // GIVEN: A report with overrides that have PASS and FAIL statuses
+        // WHEN: TextReportWriter writes the report
+        // THEN: Overrides should show original status but still appear in Exceptions section
+
+        let metadata = ReportMetadata {
+            name: None,
+            version: None,
+            url: None,
+        };
+        let mut report = ReportResults::new(1, metadata);
+
+        // Add override with PASS status
+        let override_pass_metadata = CheckerMetadata {
+            name: "test_pass".to_string(),
+            id: "1.1.1".to_string(),
+            level: 1,
+            title: "Override pass test".to_string(),
+            mode: Mode::Automatic,
+        };
+        report.add_result(
+            override_pass_metadata,
+            CheckerResult {
+                status: CheckStatus::OVERRIDE(Box::new(CheckStatus::PASS)),
+                error: String::new(),
+                override_reason: Some("Override reason for pass".to_string()),
+            },
+        );
+
+        // Add override with FAIL status
+        let override_fail_metadata = CheckerMetadata {
+            name: "test_fail".to_string(),
+            id: "2.2.2".to_string(),
+            level: 1,
+            title: "Override fail test".to_string(),
+            mode: Mode::Automatic,
+        };
+        report.add_result(
+            override_fail_metadata,
+            CheckerResult {
+                status: CheckStatus::OVERRIDE(Box::new(CheckStatus::FAIL)),
+                error: String::new(),
+                override_reason: Some("Override reason for fail".to_string()),
+            },
+        );
+
+        let writer = TextReportWriter {};
+        let mut output = Cursor::new(Vec::new());
+        writer.write(&report, &mut output).unwrap();
+
+        let output_str = String::from_utf8(output.into_inner()).unwrap();
+
+        // Verify both overrides show as SKIP with Exception mode
+        assert!(output_str.contains("[SKIP] 1.1.1     Override pass test (Exception)"));
+        assert!(output_str.contains("[SKIP] 2.2.2     Override fail test (Exception)"));
+
+        // Verify Exceptions section contains both reasons
+        assert!(output_str.contains("Exceptions:"));
+        assert!(output_str.contains("*  1.1.1 Override reason for pass"));
+        assert!(output_str.contains("*  2.2.2 Override reason for fail"));
```

Comment:
Looking at this output, it seems like overriding to a pass vs. a failure is indistinguishable. Maybe we should just insert the override value into the original bracketed text, so this would become:
```
[PASS] 1.1.1     Override pass test (Exception)
[FAIL] 2.2.2     Override fail test (Exception)
[SKIP] 3.3.3     Override skip test (Exception)

Exceptions:
*  1.1.1 Override reason for pass
*  2.2.2 Override reason for fail
*  3.3.3 Override reason for skip
```